From b116c7f25ed31f9307a82c0dada1bb2cdd1a134c Mon Sep 17 00:00:00 2001
From: TheKit <thekit@disroot.org>
Date: Fri, 28 Nov 2025 05:00:21 +0200
Subject: [PATCH 12/18] halium.config: disable protected GKI modules
 enforcement

The signatures may not match if kernel and modules come from
a different build, so relax the restrictions to make things easier
for halium-gki builds.
---
 arch/arm64/configs/halium.config | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/halium.config b/arch/arm64/configs/halium.config
index 1ac2b069d..aade174b5 100644
--- a/arch/arm64/configs/halium.config
+++ b/arch/arm64/configs/halium.config
@@ -8,6 +8,10 @@ CONFIG_DEVTMPFS=y
 CONFIG_LTO_CLANG_THIN=y
 # bluebinder
 CONFIG_BT_HCIVHCI=y
+# Do not enforce kernel modules signature
+# CONFIG_MODULE_SIG_FORCE is not set
+# CONFIG_MODULE_SIG_PROTECT is not set
+
 # AppArmor
 CONFIG_SECURITY_APPARMOR=y
 CONFIG_DEFAULT_SECURITY_APPARMOR=y
-- 
2.53.0

